FreeRTOS/FreeRTOS-Plus-TCP
# Applied PR #107 to DHCP /multi version 2

Description
-----------
It looks like PR #109 didn't get properly merged. My branch [updates_dhcp_multi](https://github.com/htibosch/FreeRTOS-Plus-TCP/tree/updates_dhcp_multi) did contain the correct code though.

This PR is based on the current contents of branch [ipv6_multi_beta](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/tree/feature/ipv6_multi_beta).

Test Steps
-----------
See earlier PR #107 .

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
